yarn.lock

```diff
@@ -1791,6 +1791,25 @@
   dependencies:
     babel-runtime "^6.26.0"
 
+"@storybook/addon-storysource@^5.3.19":
```

Comment:
Wonder if we can add something that reveals the props and let's you modify them too? I recall something like this for storybook.

Reply:
Yeah, gonna add knobs in somepoint soon. but its a little more involved